ironfish/src/wallet/wallet.ts

```diff
@@ -784,8 +766,19 @@ export class Wallet {
     }
   }
 
-  async postTransaction(raw: RawTransaction): Promise<Transaction> {
-    return this.workerPool.postTransaction(raw)
+  async postTransaction(raw: RawTransaction, memPool: MemPool): Promise<Transaction> {
+    const transaction = await this.workerPool.postTransaction(raw)
+    const verify = this.chain.verifier.verifyCreatedTransaction(transaction)
+    if (!verify.valid) {
+      throw new Error(`Invalid transaction, reason: ${String(verify.reason)}`)
+    }
+
+    await this.addPendingTransaction(transaction)
+    memPool.acceptTransaction(transaction)
+    this.broadcastTransaction(transaction)
+    this.onTransactionCreated.emit(transaction)
```

Comment:
Adding these steps here creates side effects in `postTransaction` that might break tests that use `useTxFixture`

It might be easier to split this change into a separate PR because a lot of test suites use `useTxFixture` and they'll all need to be updated for the chain to that fixture

Reply:
that is the change I am holding off to do before get agreement on the approach.